ksql-rest-app/src/main/java/io/confluent/ksql/rest/entity/CommandStatusEntity.java

```diff
@@ -27,27 +27,32 @@
 public class CommandStatusEntity extends KsqlEntity {
   private final CommandId commandId;
   private final CommandStatus commandStatus;
+  private final long commandOffset;
 
   public CommandStatusEntity(
       final String statementText,
       final CommandId commandId,
-      final CommandStatus commandStatus
+      final CommandStatus commandStatus,
+      final long commandOffset
   ) {
     super(statementText);
     this.commandId = commandId;
     this.commandStatus = commandStatus;
+    this.commandOffset = commandOffset;
   }
 
   public CommandStatusEntity(
       final String statementText,
       final String commandId,
       final String status,
-      final String message
+      final String message,
+      final String commandOffset
```

Comment:
Why a constuctor that takes a `String` for the commandOffset? It's a long, no?

If its because of the old `  public CommandStatusEntity(final Map<String, Object> properties)` constructor, then consider changing this to one that doesn't take a raw map, but allows Jackson to map the json to fields:


```
@JsonCreator
  public CommandStatusEntity(
      @JsonProperty("statementText") final String statementText,
      @JsonProperty("commandId") final String commandId,
      @JsonProperty("commandStatus") final Map<String, Object> commandStatus,
      @JsonProperty("commandOffset") final Long commandOffset
  ) {
    this(
        statementText,
        CommandId.fromString(commandId),
        new CommandStatus(
            CommandStatus.Status.valueOf(commandStatus.getOrDefault("status", ?)),
            commandStatus.getOrDefault("message", ?),
        commandOffset == null ? -1 : commandOffset
    );
  }

```
(With suitable tests proving you can serialize and deserialize)

You can then drop the one take a map.